release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.3.0.0.md

```diff
@@ -0,0 +1,15 @@
+## Version 2.3.0.0 2022-09-09
+
+Compatible with OpenSearch 2.3.0
+
+### Enhancement 
+* Change alignment of Snapshot Management panels in pages/Main/Main.tsx ([#236](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/236))
+* Update Rollup help text ([#221](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/221))
+* Refactored dependency used by test mock. Adjusted OSD version used by test workflows ([#229](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/229))
+
+### Bug fixes
+* Bumped moment version to resolve dependabot alert ([#230](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/230))
```

Comment:
This is a nitpick, but I'd probably move this to `maintenance`.

Reply:
229 or 230?

Reply:
Oh sorry, looks like the line I highlighted  isn't illustrated well here. The bullet point on line 11.

Reply:
Ah, ok let me fix that!